### PR TITLE
Organize migrations for multiple backends

### PR DESCRIPTION
**Pull request by pxp9**
Do you know a way to generate `00000000000000_diesel_initial_setup` for mysql ? 

EDIT : It does not care to have diesel initial setup for MySQL and sqlite as you can see in [diesel examples](https://github.com/diesel-rs/diesel/tree/master/examples)

### CODE REVIEW

**Comment by pxp9** [comment]
should I create `SQLite` migration ?

File: Makefile [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
-	cargo clippy --all-features
-diesel:
-	cd fang && DATABASE_URL=postgres://postgres:postgres@localhost/fang diesel migration run
-stop:
+	cargo clippy --verbose --all-targets --all-features -- -D warnings
+diesel_postgres:
+	cd fang/postgres_migrations && DATABASE_URL=postgres://postgres:postgres@localhost/fang diesel migration run
+
+diesel_mysql:
+	cd fang/mysql_migrations && DATABASE_URL=mysql://root:fang@127.0.0.1/fang diesel setup
```
Comment (Dopplerian) [inline Makefile:14]: Shouldn't this be `diesel migration run` like in the Postgres case?

Comment (pxp9) [reply Makefile]: yup fixed here

[sqlite migrations](https://github.com/ayrat555/fang/pull/134/commits/6a6f5547129037fe46c0b440262acf1081845990)

**Comment by Dopplerian** [comment]
Everything seems to be okay.

**Review by Dopplerian — APPROVED** [review]
Everything looks okay.

**Review by ayrat555 — APPROVED** [review]

**Pull request merged**